connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -281,7 +290,8 @@ private PluginScanResult scanPluginPath(
                 getPluginDesc(reflections, Connector.class, loader),
                 getPluginDesc(reflections, Converter.class, loader),
                 getPluginDesc(reflections, HeaderConverter.class, loader),
-                getPluginDesc(reflections, Transformation.class, loader)
+                getPluginDesc(reflections, Transformation.class, loader),
+                getPluginDesc(reflections, ConfigProvider.class, loader)
```

Comment:
We don't want to scan for this, but want to use the Service Loader mechanism. Do you plan to change this when the REST extension PR is merged?

Reply:
Yes, that will be for a future PR